pypeclub/OpenPype
# Slack: Added configurable maximum file size of review upload to Slack

## Brief description
Some review files might be too large to safely upload them to Slack, so configurable item in Settings set a limit.

## Description
Default is set to 50MB. If review file should be uploaded and its file size is bigger than configured, file is not uploaded and link to studio location of a file is inserted into message instead.

![Screenshot 2022-03-24 135220](https://user-images.githubusercontent.com/4457962/159920292-cda19568-5968-4e62-8d5c-e532e349941b.png)

## Testing notes:
1. configure `project_settings/slack/publish/CollectSlackFamilies/profiles/0/review_upload_limit`
2. Publish something which review file size would be bigger than configured value
3. Check message in Slack channel
